lib/tests/formulae_dependents.rb

```diff
@@ -169,8 +169,16 @@ def install_dependent(dependent, testable_dependents, args:, build_from_source:
         test "brew", "linkage", "--test", dependent.full_name
 
         if testable_dependents.include? dependent
-          test "brew", "install", "--only-dependencies", "--include-test",
-               dependent.full_name
+          test "brew", "install", "--only-dependencies", "--include-test", dependent.full_name
+
+          dependent.deps.each do |dependency|
+            next if dependency.build?
+            next if dependency.to_formula.keg_only?
+            next if dependency.to_formula.linked_keg.exist?
+
+            test "brew", "link", dependency.name
```

Comment:
I wonder if we should call `unlink_conflicts(dependency.to_formula)` before this step?

Reply:
Yep, it makes sense, will add it.